src/guest_tracking.rs

```diff
@@ -95,7 +95,11 @@ impl<T: GuestStagePagingMode> GuestVm<T> {
     pub fn new(vm: Vm<T>, page: Page<InternalClean>) -> Self {
         let page_tracker = vm.page_tracker();
         Self {
-            inner: PageArc::new_with(RwLock::new(GuestVmInner::new(vm)), page, page_tracker),
+            inner: PageArc::new_with(
+                RwLock::new(GuestVmInner::new(vm)),
+                SequentialPages::from_pages([page]).unwrap(),
```

Comment:
unwrap isn't needed here: `SequentialPages::from(page)` is infallible.

Reply:
Sure, this was temporary code of a preparation step. Nothing we'd ever run. Mostly show of future usage.